_test/test_sqw/test_PixelData.m

```diff
@@ -917,11 +1078,26 @@ function do_pixel_data_loop_with_f(obj, func, data)
 methods (Static)
 
     function advance_pix(pix, niters)
+        % Advance the pixel pages by 'niters'
```

Comment:
`niters`? That doesn't give the caller any idea what they're iterating. `n[um_]pages` would be clearer.

Reply:
assigning that as a variable in the call may make what the test is doing clearer too 
```
num_pages_to_advance = <maths>
advance_pix(pix_data, num_pages_to_advance)
```